storage/test/files.test.js

```diff
@@ -449,71 +449,109 @@ describe('storage:files', function () {
+      assert.equal(program.listFilesByPrefix.calledOnce, true);
+      assert.deepEqual(program.listFilesByPrefix.firstCall.args.slice(0, -1), [{
+        bucket: bucketName,
+        prefix: 'public/',
+        delimiter: undefined
+      }]);
     });
 
     it('should call uploadFile', function () {
       var program = getSample().program;
 
       sinon.stub(program, 'uploadFile');
       program.main(['upload', bucketName, fileName]);
-      assert(program.uploadFile.calledOnce);
+      assert.equal(program.uploadFile.calledOnce, true);
+      assert.deepEqual(program.uploadFile.firstCall.args.slice(0, -1), [{
+        bucket: bucketName,
+        srcFile: fileName
+      }]);
     });
 
     it('should call downloadFile', function () {
       var program = getSample().program;
 
       sinon.stub(program, 'downloadFile');
       program.main(['download', bucketName, fileName, fileName]);
-      assert(program.downloadFile.calledOnce);
+      assert.equal(program.downloadFile.calledOnce, true);
+      assert.deepEqual(program.downloadFile.firstCall.args.slice(0, -1), [{
+        bucket: bucketName,
+        srcFile: fileName,
+        destFile: fileName
+      }]);
     });
 
     it('should call deleteFile', function () {
       var program = getSample().program;
 
       sinon.stub(program, 'deleteFile');
       program.main(['delete', bucketName, fileName]);
-      assert(program.deleteFile.calledOnce);
+      assert.equal(program.deleteFile.calledOnce, true);
+      assert.deepEqual(program.deleteFile.firstCall.args.slice(0, -1), [{
+        bucket: bucketName,
+        file: fileName
+      }]);
     });
 
     it('should call getMetadata', function () {
       var program = getSample().program;
 
       sinon.stub(program, 'getMetadata');
       program.main(['getMetadata', bucketName, fileName]);
-      assert(program.getMetadata.calledOnce);
+      assert.equal(program.getMetadata.calledOnce, true);
+      assert.deepEqual(program.getMetadata.firstCall.args.slice(0, -1), [{
+        bucket: bucketName,
+        file: fileName
+      }]);
     });
 
     it('should call makePublic', function () {
       var program = getSample().program;
 
       sinon.stub(program, 'makePublic');
       program.main(['makePublic', bucketName, fileName]);
-      assert(program.makePublic.calledOnce);
+      assert.equal(program.makePublic.calledOnce, true);
+      assert.deepEqual(program.makePublic.firstCall.args.slice(0, -1), [{
+        bucket: bucketName,
+        file: fileName
+      }]);
     });
 
     it('should call moveFile', function () {
       var program = getSample().program;
 
       sinon.stub(program, 'moveFile');
       program.main(['move', bucketName, fileName, fileName]);
-      assert(program.moveFile.calledOnce);
+      assert.equal(program.moveFile.calledOnce, true);
+      assert.deepEqual(program.moveFile.firstCall.args.slice(0, -1), [{
+        bucket: bucketName,
+        srcFile: fileName,
+        destFile: fileName
```

Comment:
Should these `fileName`s be different? (To make sure we don't mess up the order of `srcFile` and `destFile`.)